src/OpenDebugAD7/AD7DebugSession.cs

```diff
@@ -40,6 +40,7 @@ internal class AD7DebugSession : DebugAdapterBase, IDebugPortNotify2, IDebugEven
 
         private readonly DebugEventLogger m_logger;
         private readonly Dictionary<string, Dictionary<int, IDebugPendingBreakpoint2>> m_breakpoints;
```

Comment:
>IDebugPendingBreakpoint2 [](start = 60, length = 24)

Instead of making a second map, why not just use IDebugPendingBreakpoint2.GetBreakpointRequest to check the condition?